This is not strictly necessary.
The C compiler is not used.
However, the C compiler is checked.
This causes an error if "Using the Right Compiler" is being tested (https://trac.macports.org/wiki/UsingTheRightCompiler#testing).

diff --git cmake/SuperBuild.cmake cmake/SuperBuild.cmake
index 13d6915a..5f924c81 100644
--- cmake/SuperBuild.cmake
+++ cmake/SuperBuild.cmake
@@ -231,6 +231,7 @@
 #######################################################################
 # propagate cmake variables to Netgen subproject
 set_vars( NETGEN_CMAKE_ARGS
+  CMAKE_C_COMPILER
   CMAKE_MODULE_LINKER_FLAGS
   CMAKE_MODULE_LINKER_FLAGS_RELEASE
   CMAKE_SHARED_LINKER_FLAGS
